### PR TITLE
CNV-36689: Instructions about how to log CNV bugs in Jira

### PR DESCRIPTION
**Pull request by sbeskin-redhat**
Resolves: https://issues.redhat.com/browse/CNV-36689 
Replaces https://github.com/openshift/openshift-docs/pull/73481

OCP 4.12+
CNV 4.12+

Preview: https://82835--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/support/virt-support-overview.html

### CODE REVIEW

File: virt/support/virt-support-overview.adoc [hunk 1]
```diff
@@ -6,12 +6,48 @@ include::_attributes/common-attributes.adoc[]
...
+If you have encountered an issue that requires immediate assistance from Red Hat Support, you can submit a support case.
+
+If you want to report a bug, you can create a Jira issue directly.
+
+[id="submitting-support-case_{context}"]
+=== Submitting a support case
+
+To request support from Red Hat Support, follow xref:../../support/getting-support.adoc#support-submitting-a-case_getting-support[the instructions for submitting a support case].
+
+It is helpful to collect debugging data to provide the data with your request for support.
```
Comment (jeana-redhat) [inline virt/support/virt-support-overview.adoc:23]:
```suggestion
It is helpful to collect debugging data to include with your support request.
```

File: virt/support/virt-support-overview.adoc [hunk 2]
```diff
@@ -6,12 +6,48 @@ include::_attributes/common-attributes.adoc[]
...
+Configure Prometheus and Alertmanager and collect `must-gather` data for {product-title} and {VirtProductName}.
+
+xref:../../virt/support/virt-collecting-virt-data.adoc#virt-collecting-data-about-vms_virt-collecting-virt-data[Collecting data about VMs]::
+Collect `must-gather` data and memory dumps from VMs.
+
+// must-gather not supported for ROSA/OSD, per Dustin Row
+ifndef::openshift-rosa,openshift-dedicated[]
+xref:../../virt/support/virt-collecting-virt-data.adoc#virt-using-virt-must-gather_virt-collecting-virt-data[`must-gather` tool for {VirtProductName}]::
+Configure and use the `must-gather` tool.
+endif::openshift-rosa,openshift-dedicated[]
```
Comment (jeana-redhat) [inline virt/support/virt-support-overview.adoc:40]: The first two items in this section use `must-gather`, but the "configure" link is at the end. Is this something to do first/move to the top of the list?

File: virt/support/virt-support-overview.adoc [hunk 3]
```diff
@@ -6,12 +6,48 @@ include::_attributes/common-attributes.adoc[]
...
+
+[id="virt-creating-jira-issue_{context}"]
+=== Creating a Jira issue
+
+If you want to report a bug, you can create a Jira issue directly by filling out the form on the link:https://issues.redhat.com/secure/CreateIssueDetails!init.jspa?pid=12323181&issuetype=1&priority=10200[*Create Issue*] page.
```
Comment (jeana-redhat) [inline virt/support/virt-support-overview.adoc:45]:
```suggestion
To report a bug, you can create a Jira issue directly by filling out the form on the link:https://issues.redhat.com/secure/CreateIssueDetails!init.jspa?pid=12323181&issuetype=1&priority=10200[*Create Issue*] page.
```

File: virt/support/virt-support-overview.adoc [hunk 4]
```diff
@@ -6,12 +6,48 @@ include::_attributes/common-attributes.adoc[]
...
 
 [id="virt-web-console_{context}"]
-== Web console
+== Web console monitoring
 
-The {product-title} web console displays resource usage, alerts, events, and trends for your cluster and for {VirtProductName} components and resources.
+You can monitor the health of your cluster and virtual machines (VMs) by using the {product-title} web console. The web console displays resource usage, alerts, events, and trends for your cluster and for {VirtProductName} components and resources.
```
Comment (jeana-redhat) [inline virt/support/virt-support-overview.adoc:50]: Already defined on this page above
```suggestion
You can monitor the health of your cluster and (VMs by using the {product-title} web console. The web console displays resource usage, alerts, events, and trends for your cluster and for {VirtProductName} components and resources.
```

**Review by jeana-redhat — COMMENTED** [review]
Good enhancement :)

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Comment by sbeskin-redhat** [comment]
@jeana-redhat  Thank you!

**Review by skopacz1 — APPROVED** [review]
LGTM

**Comment by skopacz1** [comment]
7

**Pull request merged**